OktaSQLiteStorage.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'OktaSQLiteStorage'
-  s.version          = '0.0.1'
+  s.version          = '0.0.2'
```

Comment:
Revert? It is not released yet

Reply:
Yes I released it to the trunk erlier.
https://github.com/CocoaPods/Specs/commit/4a563367a194b93a98c8f7d11b41ef18483805de

Reply:
got it